cluster-autoscaler/cloudprovider/gce/autoscaling_gce_client.go

```diff
@@ -88,6 +88,13 @@ var (
 	}
 )
 
+// GceInstance extends cloudprovider.Instance with GCE specific numeric id.
+type GceInstance struct {
+	Id        string
+	Status    *cloudprovider.InstanceStatus
+	NumericId uint64
+}
```

Comment:
```suggestion
type GceInstance struct {
        cloudprovider.Instance
	NumericId uint64
}
```

Reply:
This will both guard us from diverging from `cloudprovider.Instance` and simplify conversion from `GceInstance` to `cloudprovider.Instance`.